site/src/nav/Header.jsx

```diff
@@ -2,15 +2,25 @@ import DownloadButton from "./DownloadButton";
 import OvertureWordmark from "./OvertureWordmark";
 import DarkModeToggle from "./DarkModeToggle";
 import PropTypes from "prop-types";
+import GithubButton from "./GithubButton";
 
 export default function Header({ zoom, mode, setMode, setZoom }) {
   return (
     <nav aria-label="Main" className="navbar navbar--fixed-top">
       <div className="navbar__inner">
         <div className="navbar__items">
           <OvertureWordmark />
+          <a
+            aria-current="page"
+            class="navbar__item navbar__link"
```

Comment:
Oh man, this is the most minor issue but `class` should be `className`. The react code still renders with the proper classes applied, but this quiets a warning from react. 
```suggestion
            className="navbar__item navbar__link"
```